src/components/Table/GridTable.tsx

```diff
@@ -736,7 +680,7 @@ export function calcColumnSizes(columns: GridColumn<any>[], firstLastColumnWidth
         throw new Error("Beam Table column width definition only supports px, percentage, or fr units");
       }
     },
-    { claimedPercentages: 0, claimedPixels: firstLastColumnWidth ? firstLastColumnWidth * 2 : 0, totalFr: 0 },
```

Comment:
The GridDataRow element no longer takes up 100% of the table width as we now nest this element within the padding, so we do not need to account for the `firstLastColumnWidth` as part of that 100%. Maybe better explained with some examples:

Before the element containing the cells included padding elements, so we needed to account for that `firstLastColumnWidth` when coming up with these column sizes.
```
<div id=table>
  <div gridrow (width 100%)>
    <div padding />
    <div cell-1 />
    <div cell-2 />
    <div padding />
  </div>
</div>
```

Now the element containing our cell data takes up 100% of the remaining space of the table after the nested padding elements
```
<div id=table>
  <div parent-wrapper>
    <div grid-row (width 100%, which already is non-inclusive of the padding added by the parent wrapper)>
      <div cell-1 />
      <div cell-2 />
    </div>
  </div>
</div>
```

Reply:
Nice, good explanation! I'm kinda surprised... do you think this is going to cause alignment issues if a child is shown as different levels? E.g. a task three levels deep:

```
<div id=table>
  <div parent-wrapper>
  <div parent-wrapper>
  <div parent-wrapper>
    <div grid-row (width 100%, which already is non-inclusive of the padding added by the parent wrapper)>
      <div cell-1 />
      <div cell-2 />
    </div>
  </div>
  </div>
  </div>
</div>
```

Vs. one that is two levels deep:

```
```
<div id=table>
  <div parent-wrapper>
  <div parent-wrapper>
    <div grid-row (width 100%, which already is non-inclusive of the padding added by the parent wrapper)>
      <div cell-1 />
      <div cell-2 />
    </div>
  </div>
  </div>
</div>
```

Won't that inner `grid-row` have different widths between the 1st and 2nd DOMs, and so surely the cell borders are going to be thrown slightly off...

I.e. can our leaf rows can be different widths now? ... or I'm just forgetting ... in the css-grid days (of like two weeks ago:-)), the 1st and last columns were fixed down the whole table ... do we still do that?

Reply:
Oh, maybe that is what this does:

```
  const currentCardPaddingWidth =
    (style.nestedCards?.firstLastColumnWidth ?? 0) ... 
```

?

Reply:
Yes, the inner grid row would be 100% of the remaining width. But we account for that padding when drawing our row.
<img width="670" alt="Screen Shot 2022-02-08 at 9 48 41 AM" src="https://user-images.githubusercontent.com/1143861/153012689-157543f8-9ab5-4847-9dd7-5eb28a37e931.png">

Reply:
I think it'd be more difficult to do in the `wrapCard` function, because we want this `currentCardPaddingWidth` to be applied to the `leaf` card, and not on any of the open/wrapping cards.

Reply:
I think it'd be more difficult to there because we want to add this padding to the row element built by the GridRow component and not applied to the wrapping elements. Right now I pass the `row: JSX.Element` element to the `wrapCard` function. I'm not sure if there is a good way to modify that element at this point, 🤷